### PR TITLE
Mockster connector

### PR DESCRIPTION
**Pull request by miberr**
- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by miberr** [comment]
@microsoft-github-policy-service agree

**Comment by miberr** [comment]
> @miberr please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by miberr** [comment]
@microsoft-github-policy-service agree

File: certified-connectors/Mockster/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## Mockster Connector
+One time, I was training a Microsoft Official Course about Power Platform, I was asked how can one generate some testing data to Dataverse. Had no idea. I researched quite a while and couldn't find anything just right for citizen developers. There are so many mock data generators, JSON fakers, data seeders and so on. None I could find makes it easy for non-programmers to use. So Mockster was born.
```
Comment (vmanoharas) [inline certified-connectors/Mockster/README.md]: Please refer the below link and update readme.md file with all required sections. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

File: certified-connectors/Mockster/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* `Get git mockdata`: Generate git related data.
+* `Get hacker mockdata`: Generate hacker or IT related buzzwords.
+* `Get random images`: Generate random images.
+* `Get internet mockdata`: Get rich data about things in the internet like users or networking terms.
+* `Get location mockdata`: Generate location related data.
+* `Get Lorem Ipsum mockdata`: Generate lorem ipsum.
+* `Get music mockdata`: Generate music related data.
+* `Get numeric mockdata`: Generate different kinds of numbers.
+* `Get people mockdata`: Generate data for a person.
+* `Get phone mockdata`: Generate phone nuber and IMEI.
```
Comment (vmanoharas) [inline certified-connectors/Mockster/README.md]: Please correct the spelling nuber.

File: certified-connectors/Mockster/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2517 @@
...
+                    "x-ms-summary": "Phone number",
+                    "x-ms-visibility": "important"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get phone mockdata",
+        "description": "Generate phone nuber and IMEI.",
```
Comment (vmanoharas) [inline certified-connectors/Mockster/apiDefinition.swagger.json]: Please correct the spelling nuber

**Review by vmanoharas — COMMENTED** [review]
Hello @miberr,

Please resolve review comments.

File: certified-connectors/Mockster/README.md (hunk 1 above)
Comment (miberr) [reply certified-connectors/Mockster/README.md]: Hi, fixed it as the template mandated. Could you also fix the documentation at https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-a-readme-file-artifact, here the example is quite different than the template you provided.

**Comment by miberr** [comment]
All comments resolved

**Review by vmanoharas — APPROVED** [review]
Hello @miberr 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**